dash_table/Format.py

```diff
@@ -0,0 +1,200 @@
+        self.specifier['precision'] = value
+        return self
+
+    def sign(self, value: Sign = Sign.UNDEFINED):
+        value = value if isinstance(value, Sign) else Sign.UNDEFINED
+        self.specifier['sign'] = value
+        return self
+
+    def symbol(self, value: Symbol = Symbol.UNDEFINED):
+        value = value if isinstance(value, Symbol) else Symbol.UNDEFINED
+        self.specifier['symbol'] = value
+        return self
+
+    def trim(self, value: bool = False):
+        value = value if isinstance(value, bool) else False
+        value = Trim.YES if value == True else Trim.NO
+        self.specifier['trim'] = value
+        return self
+
+    def type(self, value: Type = Type.UNDEFINED):
+        value = value if isinstance(value, Type) else Type.UNDEFINED
+        self.specifier['type'] = value
+        return self
+
+    def width(self, value: int = ''):
+        value = value if isinstance(value, int) or value == '' else ''
+        self.specifier['width'] = str(value)
+        return self
+
+    def currency_prefix(self, symbol: str):
+        if 'currency' not in self.locale:
+            self.locale['currency'] = [symbol, '']
+        else:
+            self.locale['currency'][0] = symbol
+
+        return self
+
+    # Locale
+    def currency_suffix(self, symbol: str):
+        if 'currency' not in self.locale:
+            self.locale['currency'] = ['', symbol]
+        else:
+            self.locale['currency'][1] = symbol
+
+        return self
+
+    def decimal_delimitor(self, symbol: str):
+        self.locale['decimal'] = symbol
+        return self
+
+    def group_delimitor(self, symbol: str):
+        self.locale['thousands'] = symbol
+        return self
+
+    def groups(self, groups: list):
+        groups = groups if isinstance(groups, list) else \
+            [groups] if isinstance(groups, int) else None
+
+        for group in groups:
+            if not isinstance(group, int):
+                raise Exception('expected "group" to be an int')
+
+        self.locale['grouping'] = groups
+        return self
+
+    # Prefix
+    def si_prefix(self, prefix: Prefix = Prefix.UNDEFINED):
+        prefix = prefix if isinstance(prefix, Prefix) else Prefix.UNDEFINED
+        self.prefix = prefix
+        return self
+
+    def create(self):
+        f = self.locale.copy()
+        f['prefix'] = self.prefix.value
+        f['specifier'] = '{0}{1}{2}{3}{4}{5}{6}{7}{8}{9}'.format(
+            self.specifier['fill'] if self.specifier['align'] != Align.UNDEFINED else '',
+            self.specifier['align'].value,
+            self.specifier['sign'].value,
+            self.specifier['symbol'].value,
+            self.specifier['padding'].value,
+            self.specifier['width'],
+            self.specifier['group'].value,
+            self.specifier['precision'],
+            self.specifier['trim'].value,
+            self.specifier['type'].value
+        )
```

Comment:
Returns the actual format configuration object corresponding to all previous operations